### DISCUSSION TITLE
Enhanced Chest Animations

### DISCUSSION THREAD

**Discussion by deyyzee** [ThePandaOliver/Pandas-Extra-Details#33] [Ideas]
Hey! I love your mod so much, and in particular the chest animations. But I think they could still be much better.

I was thinking that when chests are open under water the chest lid could gently sway back and forth. Its such a minor detail but I think it would really add to the look of the chest.

**Comment by ThePandaOliver** [comment]
Great idea, i will remember that, maybe i should make all blocks animations slower when they are waterlogged

**Comment by deyyzee** [comment]
That would be great. Although I think that would only apply to the chest, trapdoor, and hanging sign animations.

Also, underwater swaying animations would look cool when applied to open trapdoors.
